app/translations/es/LC_MESSAGES/messages.po

```diff
@@ -945,3 +943,20 @@ msgstr ""
 #~ msgid "Invoice Footer"
 #~ msgstr ""
 
+#~ msgid ""
+#~ "We\n"
+#~ "hope you enjoyed it"
+#~ msgstr ""
+
+#~ msgid "Click on the following link to manage your event"
```

Comment:
The messages in this area are half sentences. Please make the whole sentences translatable instead of only parts. Always include the full stop in the translatable string itself.